src/tdsops.f90

```diff
@@ -38,15 +38,14 @@ module m_tdsops
   end interface tdsops_t
 
   type :: dirps_t
-      !! Directional tridiagonal solver container.
-      !!
-      !! This class contains the preprocessed tridiagonal solvers for operating
-      !! in each coordinate direction.
-    class(tdsops_t), allocatable :: der1st, der1st_sym, &
-      der2nd, der2nd_sym, &
-      stagder_v2p, stagder_p2v, &
-      interpl_v2p, interpl_p2v
+    !! Directional tridiagonal solver container.
+    !!
+    !! This class contains the preprocessed tridiagonal solvers for operating
+    !! in each coordinate direction.
+    class(tdsops_t), allocatable :: der1st, der1st_sym, der2nd, der2nd_sym, &
+      stagder_v2p, stagder_p2v, interpl_v2p, interpl_p2v
     integer :: nrank, nproc, pnext, pprev, n, n_blocks, dir
```

Comment:
The `nrank` and `nproc` here meant to be the directional `nrank` and directional `nproc`, but I see that it might be confusing. Maybe its better renaming them `nrank_dir`, `nproc_dir`?

Reply:
I think the suggested change is good